pkgs/development/haskell-modules/generic-builder.nix

```diff
@@ -222,7 +253,15 @@ assert allPkgconfigDepends != [] -> pkgconfig != null;
 stdenv.mkDerivation ({
   name = "${pname}-${version}";
 
-  outputs = [ "out" ] ++ (optional enableSeparateDataOutput "data") ++ (optional enableSeparateDocOutput "doc");
+  outputs =
```

Comment:
Hmm, no `dev` output? I would expect one to contain all the `.o` and `.hi` files, plus the GHC package `.conf` file.